bowtie/_report.py

```diff
@@ -314,8 +314,20 @@ def flat_results(self):
     def generate_badges(self, target_dir: Path, dialect: str):
         label = _DIALECT_URI_TO_SHORTNAME[dialect]
         for impl in self.implementations:
+            dialectVersions = impl["dialects"]
             if dialect not in impl["dialects"]:
                 continue
+
+            SupportedVersions: str = []
```

Comment:
You'll see CI is failing, because this type hint is wrong, you're saying `supported_versions` is a string, but it isn't, it's a `list[str]`. But all of these lines are spelled `str.join` in Python -- so you want simply:

```suggestion
            supported = ", ".join(
                each.removeprefix("Draft ") for each in dialect_versions
            )
```